src/credentials/BUILD.gn

```diff
@@ -199,4 +200,6 @@ static_library("test_dac_revocation_delegate") {
     ":credentials",
     jsoncpp_root,
   ]
+
+  # deps = [ dir_pw_unit_test ]
```

Comment:
is it needed and not enabled or was it part of some test?

Reply:
Nice find! ... I thought it was needed however in the end it appears not to be. Will remove. Thank you for spotting this!